rareapi/views/PostTags.py

```diff
@@ -0,0 +1,60 @@
+from django.core.exceptions import ValidationError
+from rest_framework.viewsets import ViewSet
+from rest_framework import serializers, status
+from rest_framework.response import Response
+from rareapi.models import PostTags, Tags, Posts
+
+class PostTagsViewSet(ViewSet):
+    def list(self, request):
+
+        posttags = PostTags.objects.all()
+
+        post_id = self.request.query_params.get("postId", None)
+        if post_id is not None:
+            posttags = posttags.filter(post_id=post_id)
+        
+        serializer = PostTagsSerializer(posttags, many=True, context={'request', request})
+        return Response(serializer.data)
+
+    def create(self, request):
+        post_id = request.data["post_id"]
+        try:
```

Comment:
This validation logic totally works.... but I feel like nesting the `try` statements like this so heavily makes it definitely harder to read for me... But that's just my opinion it all still works